lib/generators/annotate/templates/auto_annotate_models.rake

```diff
@@ -13,6 +13,16 @@ if Rails.env.development?
       'position_in_fixture'     => 'before',
       'position_in_factory'     => 'before',
       'position_in_serializer'  => 'before',
+      'include_model'           => true,
```

Comment:
Heh, what you're suggesting here is exactly how I first implemented it. I changed it later to be more consistent with existing options (like `exclude_*`). I'll go back to the previous approach.